### PR TITLE
Fix crash in "Insert at video position"

### PR DESCRIPTION
**Pull request by OmrSi**
Introduced in https://github.com/SubtitleEdit/subtitleedit/pull/4385

### CODE REVIEW

File: src/Forms/Main.cs [hunk 1]
```diff
@@ -19228,7 +19228,7 @@ private Paragraph InsertNewTextAtVideoPosition()
 
             // prevent overlap
             var endTotalMilliseconds = videoPositionInMilliseconds + Configuration.Settings.General.NewEmptyDefaultMs;
-            var next = _subtitle.Paragraphs[index];
+            var next = _subtitle.Paragraphs.ElementAtOrDefault(index);
```
Comment (niksedk) [inline src/Forms/Main.cs:19231]: SE uses `_subtitle.GetParagraphOrDefault`

Comment (OmrSi) [reply src/Forms/Main.cs:19231]: And I messed up the word "Paragraph" in the new commit. smh

Comment (OmrSi) [reply src/Forms/Main.cs:19231]: This is better than `count > index`, right?

Comment (OmrSi) [reply src/Forms/Main.cs:19231]: Let me know if it's better to close this and open a new one.
Or you might wanna do the commit yourself and close this. :D

**Comment by OmrSi** [comment]
@niksedk Just learned about "force push".
I'll use it next time...

**Pull request merged**